ModelViewer/ModelViewer/ModelLoader.cpp

```diff
@@ -72,8 +73,32 @@ Mesh ModelLoader::ProcessMesh(aiScene const* pScene, uint meshIdx)
 
 
 
-	// TODO: Handle textures
-
+	// Search for an ambient, diffuse, or specular texture.  We will only load the first one we find because our shader
+	// only handles a single texture
+	std::vector<aiTextureType> validTextureTypes = { aiTextureType_AMBIENT, aiTextureType_DIFFUSE, aiTextureType_SPECULAR };
+	newMesh.m_hasTexture = false;
+	for (auto it : validTextureTypes)
+	{
+		// Only load 1 texture
+		if (newMesh.m_hasTexture) {
```

Comment:
Do we want to prompt the user to select a valid texture? Might be a 'sprint 4' kind of issue XD

Reply:
I'm not really a fan of user prompts (slows down our quick loading).  This method is very likely to load the correct type of texture for what our shader is capable of.  The other texture types are like emissive, normals, and bumpmapped stuff.

Reply:
In the future, we could support all that, but it would be a major pain :)